Cargo.toml

```diff
@@ -27,6 +27,7 @@ thiserror = "1.0"
 toml = "0.5"
 yn = "0.1"
 rnix = "0.8"
+smol_str = "=0.1.16" # required by rnix, but 0.1.17 doesn't build on nixos-20.09
```

Comment:
I'd like the note to include the Rust version of NixOS 20.09 as well as the Rust version which is new enough to not have this issue

Reply:
Good point, I'll add it.